docs/airflow/static-schedule-pipeline.md

```diff
@@ -27,3 +27,4 @@ gtfs_loader | gtfs_downloader, gtfs_schedule_history | Daily | No | Loads and pa
 gtfs_schedule_history2 | gtfs_loader | Daily | Yes | Analyzes the current day's result and creates data about any changes in feed contents
 gtfs_views_staging | gtfs_schedule_history2 | Daily | No | Creates staging tables for various view (Dimension and Fact) tables
 gtfs_views | gtfs_views_staging | Daily | No | Creates view (Dimension and Fact) tables
+rt_loader | | Daily | No | Parses and loads GTFS RT data
```

Comment:
Sorry this is incredibly nitpicky but I don't think we want to add this on this page since this page is dedicated to the static pipeline.... Not sure if we want a separate docs page for the RT pipeline? But I don't think this belongs here, unless we're making this a more general GTFS page. Defer to @charlie-costanzo

Reply:
Probably the right call to not include the docs change in this section. Good catch, @lauriemerrell.

Reply:
Oops definitely just added it where I saw other DAG docs.

@charlie-costanzo are you OK with not documenting this DAG for now?